plugins/dra-ocp-validator/tests/test-dra-extended-resources.sh

```
@@ -0,0 +1,215 @@
+#!/bin/bash
+# DRA Extended Resources Validation Test
+# Uses existing NVIDIA DeviceClasses created by the driver
+
+set -e
+
+NAMESPACE="dra-extended-test"
+LOG_DIR="./dra-extended-validation-$(date +%Y%m%d-%H%M%S)"
+mkdir -p ${LOG_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)